src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/WatchdogLeases.sql

```diff
@@ -0,0 +1,14 @@
+﻿--DROP TABLE WatchdogLeases
+GO
+CREATE TABLE dbo.WatchdogLeases
+  (
+       Watchdog            varchar(100)  NOT NULL
+      ,LeaseHolder         varchar(100)  NOT NULL CONSTRAINT DF_WatchdogLeases_LeaseHolder DEFAULT ''
+      ,LeaseEndTime        datetime      NOT NULL CONSTRAINT DF_WatchdogLeases_LeaseEndTime DEFAULT 0
```

Comment:
Should timers record when the last instance was executed? This would give you something to resume from is something happened to the original process

Reply:
In general case, the state of the abandoned process cannot be determined by its last success date. It might be useful in certain cases, but not generic...

Reply:
This doesn't guarantee that a timer will be run at a regular interval though. Timers that are run less often could more likely be impacted by abandoned processes, or be subject to running at more or less frequent intervals than intended